src/rofl/datapath/pipeline/openflow/openflow1x/pipeline/of1x_flow_table.h

```diff
@@ -172,6 +172,13 @@ rofl_result_t __of1x_destroy_table(of1x_flow_table_t* table);
 */
 
 /**
+* @internal Add flow entry to table being able to match cookie
+*
+* @warning Out of spec.
```

Comment:
can you elaborate more on the problems that could happen, in case you want to use this still?

I guess you could end up with flow entries, that have the same matches, right? So it would be nice to highlight the correct use.